Steer/DigitizerWorkflow/src/CTPDigitWriterSpec.h

```diff
@@ -0,0 +1,34 @@
+//
+// Created by rl on 3/19/21.
+//
+
+#ifndef STEER_DIGITIZERWORKFLOW_CTPDIGITWRITERSPEC_H
+#define STEER_DIGITIZERWORKFLOW_CTPDIGITWRITERSPEC_H
+
+#include "Framework/DataProcessorSpec.h"
+#include "DPLUtils/MakeRootTreeWriterSpec.h"
+#include "Framework/InputSpec.h"
+#include "DataFormatsCTP/Digits.h"
+
+namespace o2
+{
+namespace ctp
+{
+
+template <typename T>
+using BranchDefinition = framework::MakeRootTreeWriterSpec::BranchDefinition<T>;
+
+o2::framework::DataProcessorSpec getCTPDigitWriterSpec(bool mctruth)
+{
+  using InputSpec = framework::InputSpec;
+  using MakeRootTreeWriterSpec = framework::MakeRootTreeWriterSpec;
+  return MakeRootTreeWriterSpec("CTPDigitWriter",
+                                "CTPdigits.root",
```

Comment:
usually digitization produces files with lower-case names, better to stick to this.